src/Microsoft.AspNetCore.Server.IISIntegration/Server/IISHttpContextOfT.cs

```diff
@@ -19,10 +19,10 @@ public IISHttpContextOfT(PipeFactory pipeFactory, IHttpApplication<TContext> app
             _application = application;
         }
 
-        public override async Task ProcessRequestAsync()
+        public override async Task<bool> ProcessRequestAsync()
```

Comment:
Is this change related? Why isn’t this another PR?

Reply:
It is related. If the request is aborted, we would like to return FINISH_REQUEST to the IIS pipeline. Unless you want to go with the logic of always returning continue.

Reply:
Is that what’s happening here? I don’t see it. Where does abort cause success to be false?

Reply:
Calling abort could throw an exception from user code on writing (or other operations on the context).

Reply:
What does that have to do with ending the request? The user code can catch the exception.

Reply:
https://github.com/aspnet/IISIntegration/pull/469